src/components/datepicker/DatePickerInput.js

```diff
@@ -1,20 +1,20 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-import Box, { omitBoxProps, pickBoxProps } from '../box';
-import DayPickerInput from 'react-day-picker/DayPickerInput';
+import Box, { pickBoxProps } from '../box';
+import DatePicker from '../datepicker';
 import Icon from '../icon';
-import NavigationBar from './NavigationBar';
-import WeekDay from './WeekDay';
-import { convertModifiersToClassnames } from './utils';
-import { IconCalendarSmallOutline } from '@teamleader/ui-icons';
-import ValidationText from '../validationText';
+import Input from '../input';
+import Popover from '../popover';
 import cx from 'classnames';
-import omit from 'lodash.omit';
 import theme from './theme.css';
+import MomentLocaleUtils from 'react-day-picker/moment';
```

Comment:
This on its turn imports the entire `moment` library. No go 👎 
https://github.com/gpbl/react-day-picker/blob/master/src/addons/MomentLocaleUtils.js#L3

Reply:
I know, this `react-day-picker` package relies on the `Moment` library. I previously tried to convert these local utils to work with Luxon.
But, the problem is with the `getFirstDayOfWeek` function, which decides if the locale first day of the week is a monday or sunday. `Luxon` (based on the Intl api) or `Date-fns` doesn't provide such a function, forcing us to keep using Moment under the hood of our DatePickerInput component.

Before this PR we had to pass the `MomentLocaleUtils` as a prop. But since we don't have a solid alternative, I decided to build this inside our `DatePickerInput` component.

Unless anyone has a better solution, I think we should merge this PR :)

**Sources:**
https://github.com/moment/luxon/issues/373
https://stackoverflow.com/questions/50350110/how-to-create-momentlocaledata-firstdayofweek-in-luxon

Reply:
And we can't provide the `getFirstDayOfWeek` function ourselves in any way? Importing the entire `momentjs` package is not really acceptable as it increases our package size a lot.

Reply:
@lowiebenoot You mean by hard-coding the locales (string) and their first day (integer) of the week?

Something like this?

```
const firstDayOfWeek = {
  fr: 1,
  nl: 1,
  pt: 0,
...
};

const getFirstDayOfWeek = locale => {
  return firstDayOfWeek[locale];
}
```

Reply:
That's also basically how moment does it :)

Reply:
Yeah, I guess. It has to be "hard-coded" somewhere in the end... It's not that you can derive it from the language so that code has to be somewhere.